index.html

```diff
@@ -5502,7 +5504,101 @@ <h2>Base Context</h2>
   }
 }
 </pre>
+      </section>
+
+      <section class="informative">
+        <h3>Differences between Contexts, Types, and CredentialSchemas</h3>
 
+        <p>
+The verifiable credential and verifiable presentation data models leverage a
+variety of underlying technologies including [JSON-LD] and [JSON-SCHEMA-2018].
+This section will provide a comparison of the <code>@context</code>,
+<code>type</code>, and <code>credentialSchema</code> properties, and cover some
+of the more specific use cases where it is possible to use these features of the
+data model.
+        </p>
+        
+        <p>
+The primary purpose of the <code>type</code> property is to uniquely
+identify the type of the <code>verifiableCredential</code> in which it
+appears i.e. to indicate which set of claims the
+<code>verifiableCredential</code> contains. This property is mandatory and
+the value "VerifiableCredential" must appear in the set of values. Whilst
+it is good practice to include one additional value depicting the unique
+subtype of this verifiableCredential, it is not forbidden to either omit
+additional type values or to include multiple type values in the array.
+Since many verifiers will request verifiableCredentials of a specific
+subtype, then omitting the subtype value may make it more difficult for
+verifiers to inform the holder which verifiableCredential they require.
+When a verifiableCredential is of multiple subtypes, then listing all of
+them in the type property is sensible. However, the usage of the
+<code>type</code> property in a JSON-LD representation of a credential
+conveys more than what it means in a JSON representation of a credential.
```

Comment:
This is a bit strange... the semantics are the same between JSON and JSON-LD. It's just that JSON-LD is actually capable of enforcing the semantics, where JSON cannot.

Reply:
That's the problem I see with this. By saying that only the JSON-LD version can enforce things it opens the possibility of miss attribution by a developer in a way that could cause issues. I'm open to rewording this if you'd like. Could you provide some suggested text?

Reply:
updated the text to make this a bit more clear hopefully.